client/src/components/Message/MessageList.tsx

```diff
@@ -1,18 +1,32 @@
 import React from "react";
+import styled from "styled-components";
 import { Member, Message as IMessage } from "graphql/generated/graphql";
 
 import Message from "components/Message/Message";
 import { useAuthContext } from "contexts/AuthContext";
 
 interface IMessageList {
   messages?: IMessage[];
+  onScroll?: (event: any) => void;
 }
 
-const MessageList: React.FC<IMessageList> = ({ messages }) => {
+const StyledMessageList = styled.section`
+  overflow-y: scroll;
+  height: calc(100vh - 150px);
```

Comment:
Fixes the layout of Messages and scroll overflow.... -150px to offset the <RoomHeader /> height